### PR TITLE
Feature/checkout categories

### PR DESCRIPTION
**Pull request by JeanVittory**
- Ending up the reading process of appointments at cart to can show them in checkout page

- Writing up a redux method to delete a specific appointment throughout an id provided from the checkout component

### CODE REVIEW

File: src/pages/appointments/adapters/doctorsByArea.adapters.js [hunk 1]
```diff
@@ -5,13 +5,15 @@ export const doctorsByAreaAdapter = (response) => {
         id: area.id,
```
Comment (ht1204) [inline src/pages/appointments/adapters/doctorsByArea.adapters.js:5]: you can destructure the area object item:
```javascript
return response.map(({id, area, doctors, price}) => {
``` 

Would be better yet.

**Review by ht1204 — COMMENTED** [review]
I'm attentive to any updates according this PR, you had a merge conflict, I proposed a solution, please, pull these changes in your local branch, test them in order to avoid issues.

File: src/pages/appointments/adapters/doctorsByArea.adapters.js (hunk 1 above)
Comment (JeanVittory) [reply src/pages/appointments/adapters/doctorsByArea.adapters.js]: Hi Hector, your recommendation was implemented. I extend it putting the destructuring at every adapter with less of 4 values coming. Was corrected some others little bugs and typos.

Comment (ht1204) [reply src/pages/appointments/adapters/doctorsByArea.adapters.js]: Awesome, thanks, keep doing the good job.

**Review by ht1204 — APPROVED** [review]

**Pull request merged**